raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -58,7 +58,12 @@ library ChannelManagerLibrary {
 
         address[] storage existingChannels = self.nodeChannels[msg.sender];
```

Comment:
I don't think it's an issue. There can only ever exist one contract for two given addresses. This means that if we get a hit and delete an element, we don't need to iterate through the rest of the list. Or maybe I misunderstand what you mean.